### PR TITLE
UserStory #45 - Edit business info

### PR DESCRIPTION
**Pull request by therobotisnotatoy**
Updated for better integration with UserStory #18 Create business and Feature #154 Tests for PUT/PATCH method for editing business info

### CODE REVIEW

File: beauty/api/permissions.py [hunk 1]
```diff
@@ -24,33 +26,21 @@ def has_object_permission(self, request, view, obj):
...
-        """Object permission check."""
-        logger.debug(f"Object {obj.id} permission check")
-
-        if request.user.is_authenticated:
-            return request.user.is_admin or (obj == request.user)
+        """Verify that the current user is selected business owner or administrator."""
+        logger.info(f"User {request.user} permission check")
+        try:
+            request.user.is_admin
+            return obj.owner == request.user
```
Comment (morento101) [inline beauty/api/permissions.py]: Add check if a user is an admin

**Review by morento101 — COMMENTED** [review]
fix permission and I will approve:innocent:

File: beauty/api/permissions.py [hunk 2]
```diff
@@ -2,6 +2,8 @@
```
Comment (pazuzu-ua) [inline beauty/api/permissions.py:2]: application - spelling

**Review by pazuzu-ua — APPROVED** [review]

**Review by Misha86 — APPROVED** [review]

File: beauty/api/permissions.py (hunk 1 above)
Comment (therobotisnotatoy) [reply beauty/api/permissions.py]: Done

File: beauty/api/permissions.py (hunk 2 above)
Comment (therobotisnotatoy) [reply beauty/api/permissions.py:2]: Reread three times until I realized where the spelling is

File: beauty/beauty/urls.py [hunk 3]
```diff
@@ -26,7 +26,7 @@
 from rest_framework.routers import DefaultRouter
 from beauty.yasg import urlpatterns as doc_urls
 
-from api.views_api import (ResetPasswordView, UserActivationView, UserViewSet)
+from api.views import (ResetPasswordView, UserActivationView, UserViewSet)
```
Comment (morento101) [inline beauty/beauty/urls.py]: Does it not throw an error? Views is a folder, not a module

**Review by morento101 — APPROVED** [review]
Great!!!

**Review by datsiuk-ira — APPROVED** [review]

File: beauty/beauty/urls.py (hunk 3 above)
Comment (therobotisnotatoy) [reply beauty/beauty/urls.py]: done

**Pull request merged**